api/src/main/java/org/ehcache/exceptions/SerializerException.java

```diff
@@ -18,24 +18,44 @@
 
 /**
  * Exception thrown by a {@link org.ehcache.Cache} when a store cannot serialize or deserialize a value.
+ *
  * @author Ludovic Orban
  */
 public class SerializerException extends RuntimeException {
 
   private static final long serialVersionUID = -4008956327217206643L;
 
+  /**
+   * No args constructor for this exception.
+   */
   public SerializerException() {
```

Comment:
god kills a kitten every time we write javadoc to satisfy checkstyle for obvious things like this